### DISCUSSION TITLE
version 0.54.0 cant install plugins

### DISCUSSION THREAD

**Discussion by lozanotux** [aquasecurity/trivy#7287] [Bugs]
### Description

I'm trying to install new plugins using names, urls and files. But always I got HTTP 404 error:

![image](https://github.com/user-attachments/assets/cf868d77-3f8a-4ec0-8589-5e459dd511f2)

But when I used 0.53.0 version all goes ok:

![image](https://github.com/user-attachments/assets/22b46f01-10ed-4a1f-9508-39b7d7d81a9b)

### Desired Behavior

When I use `trivy plugin install` command with names, urls or files. The plugin must be installed

### Actual Behavior

I use `trivy plugin install` command but I get the error `HTTP 404 failed to round trip: failed to get the file content`

### Reproduction Steps

1. Execute a container with trivy: 
```bash
podman run -it --rm --name trivy --user root --entrypoint="/bin/bash" bitnami/trivy:0.54.0
```

2. Verify trivy version:
```bash
trivy -v
```

3. Install any plugin:
```bash
trivy plugin install sonarqube
```

### Target

Container Image

### Scanner

Vulnerability

### Output Format

JSON

### Mode

Standalone

### Debug Output

```bash
$ trivy plugin install https://github.com/umax/trivy-plugin-sonarqube/releases/download/v0.2.2/trivy-sonarqube.tar.gz --debug
2024-07-31T17:23:47Z	DEBUG	Cache dir	dir="/.cache/trivy"
2024-07-31T17:23:47Z	INFO	[plugin] Updating the plugin index...	url="https://aquasecurity.github.io/trivy-plugin-index/v1/index.yaml"
2024-07-31T17:23:47Z	INFO	[plugin] Installing the plugin...	src="https://github.com/umax/trivy-plugin-sonarqube/releases/download/v0.2.2/trivy-sonarqube.tar.gz"
2024-07-31T17:23:48Z	FATAL	Fatal error
  - plugin install error:
    github.com/aquasecurity/trivy/pkg/commands.NewPluginCommand.func2
        /bitnami/blacksmith-sandox/trivy-0.54.0/src/github.com/aquasecurity/trivy/pkg/commands/app.go:768
  - download failed:
    github.com/aquasecurity/trivy/pkg/plugin.(*Manager).install
        /bitnami/blacksmith-sandox/trivy-0.54.0/src/github.com/aquasecurity/trivy/pkg/plugin/manager.go:116
  - download error:
    github.com/aquasecurity/trivy/pkg/downloader.DownloadToTempDir
        /bitnami/blacksmith-sandox/trivy-0.54.0/src/github.com/aquasecurity/trivy/pkg/downloader/download.go:49
  - failed to download https://github.com/umax/trivy-plugin-sonarqube/releases/download/v0.2.2/trivy-sonarqube.tar.gz:
    github.com/aquasecurity/trivy/pkg/downloader.Download
        /bitnami/blacksmith-sandox/trivy-0.54.0/src/github.com/aquasecurity/trivy/pkg/downloader/download.go:99
  - Get "https://github.com/umax/trivy-plugin-sonarqube/releases/download/v0.2.2/trivy-sonarqube.tar.gz": failed to round trip: failed to get the file content: GET https://api.github.com/repos/umax/trivy-plugin-sonarqube/contents/releases/download/v0.2.2: 404 Not Found []
```

### Operating System

Debian GNU/Linux 12 (bookworm)

### Version

```bash
$ trivy --version
Version: 0.54.0
```

### Checklist

- [X] Run `trivy clean --all`
- [X] Read [the troubleshooting](https://aquasecurity.github.io/trivy/latest/docs/references/troubleshooting/)

**Comment by nikpivkin** [comment]
Hi @lozanotux !

A fix for this [issue](https://github.com/aquasecurity/trivy/issues/7273) will be in version 54.1 .

**Comment by nikpivkin** [comment]
Duplicate of https://github.com/aquasecurity/trivy/issues/7273
